doc/man3/X509_get_subject_name.pod

```diff
@@ -25,16 +34,29 @@ subject names
 
 =head1 DESCRIPTION
 
+X509_NAME_hash_ex() returns the SHA1 hash value of name B<x> or 0 on failure,
```

Comment:
`I<x>`

I know this is inconsistent with the unmodified text. But you're using `I<libctx>` anyway. I am not forcing you to change all the wrong occurrences in the file.

Reply:
Ah, right; I should better have made all of them consistently use `I<`...`>`; done now.